docker-compose.yml

```diff
@@ -122,6 +122,37 @@ services:
     labels:
       logs: "promtail"
 
+  hyperswitch-web:
+    ports:
+      - "9050:9050"
+      - "9060:9060"
+      - "5252:5252"
+    build:
+      context: ./docker
+      dockerfile: web.Dockerfile
+    networks:
+      - router_net
+    environment:
+      - HYPERSWITCH_SERVER_URL=http://localhost:8080
```

Comment:
Will this container be able to interact with the server at `localhost:8080`? We would need some more changes and/or the URL might have to be updated?

Reply:
It was working fine when tested. Still Updated with the changes required.